README.md

```diff
@@ -1,14 +1,16 @@
-# Samba-Gluster Integration Test Environment
+# Samba-Filesystem Integration Test Environment
```

Comment:
Sorry to comment again.

Should we really mention Filesystem in the header itself? Are we good with **Samba Integration Test Environment**?

Reply:
That should be fine.
 Samba-Filesystem Integration is more of a word to word translation.
 Samba Integration sounds good as a phrase. I can update it